client/task-list/tasks/payments/RemotePayments/components/RecommendedPaymentGatewayList/RecommendedPaymentGatewayList.js

```diff
@@ -1,122 +1,51 @@
 /**
  * External dependencies
  */
-import classnames from 'classnames';
-import { Fragment } from '@wordpress/element';
-import {
-	Card,
-	CardBody,
-	CardMedia,
-	CardHeader,
-	CardDivider,
-} from '@wordpress/components';
-import { Text } from '@woocommerce/experimental';
-import { recordEvent } from '@woocommerce/tracks';
+import { Card, CardHeader } from '@wordpress/components';
+import { useMemo } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { PaymentAction } from '../../../components/PaymentAction';
-import { RecommendedRibbon } from '../../../components/RecommendedRibbon';
-import { SetupRequired } from '../../../components/SetupRequired';
+import { RecommendedPaymentGatewayListItem } from './RecommendedPaymentGatewayListItem';
 
 import './RecommendedPaymentGatewayList.scss';
 
+const RECOMMENDED_GATEWAY_KEYS = [ 'wcpay', 'mercadopago', 'stripe' ];
```

Comment:
There's an issue with having this logic in this component instead of the root `RemotePayments/index` file. The `recommendedPaymentGateways` array will generally not contain `wcpay` here, since it's sidelined into the WCPay card component until it's fully enabled and configured. Right now this could result in more than one gateway being recommended.

Reply:
Very true.  Moved back up to the remote payments index. 👍